Source/Editor/Modules/UIModule.cs

```diff
@@ -382,7 +383,7 @@ private void InitMainMenu(RootControl mainWindow)
             MainMenu.Parent = mainWindow;
 
             // File
-            MenuFile = MainMenu.AddButton("File");
+            MenuFile = MainMenu.AddButton(EditorLocalization.Manager.GetValue("T_CM_FILE"));
```

Comment:
We could use some utility to make those texts easier to replace, eg.
```cs
Loc.T("T_CM_FILE")
```

Reply:
For what it's worth, one really funky alternative that I've seen (ab)uses extension methods. e.g.
```cs
"T_CM_FILE".Loc()
```